rke/structure_rke_cluster.go

```diff
@@ -293,11 +293,11 @@ func expandRKECluster(in *schema.ResourceData) (string, *rancher.RancherKubernet
 		obj.DNS = expandRKEClusterDNS(v)
 	}
 
-	if v, ok := in.Get("enable_cri_dockerd").(bool); ok {
+	if v, ok := in.Get("enable_cri_dockerd").(bool); ok && obj.EnableCRIDockerd == nil {
```

Comment:
Note to other reviewers:  
In the case of importing an RKE cluster, we define an empty rke_cluster resource in the terraform configuration file:
```
resource "rke_cluster" "test1" {
  # (resource arguments)
}
```
The value for `enable_cri_dockerd` is read from the provided cluster.yml file,  not specified in the above rke_clutster resource.  Therefore, the value should not be overwritten if it is not nil. 

The same rule is applied to the `ignore_docker_version` field  below.